doc/yjit/yjit.md

```diff
@@ -242,6 +241,56 @@ Performance counter names are not guaranteed to remain the same between Ruby ver
 
 The printed text after a --yjit-stats run includes other information that may be named differently than the information in runtime_stats.
 
+## Speeding Up YJIT
```

Comment:
```suggestion
## Performance Tips for Production Deployments
```

Reply:
applied in https://github.com/ruby/ruby/pull/8472/commits/e972db4aa3ef32c10ab085bd5abf1baea8de2b62